### PR TITLE
PR and Issue Templates

### PR DESCRIPTION
**Pull request by awwad**
This adds template files in a hidden directory, .github. The result will be that when a user creates an Issue or a Pull Request, there will be default text populating the fields.

This should encourage more descriptive PRs.

See these how-tos:
- https://help.github.com/articles/creating-an-issue-template-for-your-repository/
- https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

### CODE REVIEW

**Comment by SantiagoTorres** [comment]
Hey! I only see a PR template here. Maybe you forgot to git add?

**Comment by SantiagoTorres** [comment]
Also, quick followup, would it be a good idea to host these templates in @secure-systems-lab?

**Comment by awwad** [comment]
I haven't added an issue template to the PR yet. Trying to decide between a user-oriented issue template and a developer-oriented issue template.

Yeah, there should be templates in that repo, too, and it would be good to use those as models in other repos.

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Fixes #.
```
Comment (vladimir-v-diaz) [inline .github/PULL_REQUEST_TEMPLATE]: Should we have more sections?

I think having sections for the "purpose of the pull request", "summary of the changes", "Usage/example of changes", etc. are helpful to reviewers.

Reference:
https://github.com/blog/1943-how-to-write-the-perfect-pull-request

**Review by vladimir-v-diaz — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (awwad) [reply .github/PULL_REQUEST_TEMPLATE]: Good suggestion. Added some in 243e69c

**Comment by awwad** [comment]
I'd like to just merge this, but it's always good to get another eye. Look OK enough to you, @vladimir-v-diaz?

**Comment by vladimir-v-diaz** [comment]
LGTM.

Even better if we can provide an example of a good issue report/request in the template.  I guess you can either provide a link, or include an example in the template itself (which the user/developer would replace)

**Comment by vladimir-v-diaz** [comment]
For the bug report case, we should consider having another heading for the actual problem.  The developer/user should describe the bug or issue, as well.

**Pull request merged**